class/defaults.yml

```
@@ -1,3 +1,54 @@
 parameters:
   rancher_monitoring:
     namespace: syn-rancher-monitoring
+
+    kube_prometheus_version:
+      '1.15': 53f18a42769ea1d4bce833962a908f84069ef495
+      '1.16': 68692f7aec038090bfba951bdf821dd8f1c08e75
+      '1.17': 4e7440f742df31cd6da188f52ddc4e4037b81599
+      '1.18': f69ff3d63de17f3f52b955c3b7e0d7aff0372873
+    cluster_kubernetes_version: '1.18'
+    jsonnetfile_parameters:
+      kube_prometheus_version: ${rancher_monitoring:kube_prometheus_version:${rancher_monitoring:cluster_kubernetes_version}}
+
+    prometheusInstance: platform
+    prometheus:
+      storage:
+        volumeClaimTemplate:
+          spec:
+            storageClassName: fast
+            resources:
+              requests:
+                storage: 10Gi
+
+    alertmanagerInstance: platform
+    alertmanager:
+      replicas: 1
+      logLevel: info
+
+    # alertmanagerConfig is rendered as is into the alertmanager config secret
+    alertmanagerConfig:
+      receivers:
+        # blackhole receiver for alerts we don't care about
+        - name: devnull
+      # If the user doesn't configure a receiver, send everything to
+      # devnull. This allows Alertmanager to start even if the user doesn't
+      # configure any alert receivers.
+      route:
+        receiver: devnull
```

I'd propose to set the following additional defaults:
```suggestion
        receiver: devnull
        group_wait: 0s
        group_interval: 5s
        repeat_interval: 10m
```